test/spec/services/osm_wikibase.js

```diff
@@ -277,12 +277,12 @@ describe('iD.serviceOsmWikibase', function () {
         }),
         status: 200,
         headers: { 'Content-Type': 'application/json' }
-    });
+      });
 
       wikibase.getEntity({ key: 'amenity', value: 'parking', langCodes: ['fr'] }, callback);
 
       await setTimeout(50);
-      expect(parseQueryString(fetchMock.calls()[0][0])).to.eql(
+      expect(parseQueryString(fetchMock.calls(/action=wbgetentities/)[0][0])).to.eql(
```

Comment:
there might be other services calling API requests, so explicitly specify which mock we're interested in